BAH-1887| Samar | Updated simulation, code clean up

samar-tw opened the pull request:
* Simulations are set according to updated Gatling version
* Kept maven plugin for Gatling to 3.0.5 as mvn:test can be executed from CI. Latest was having issue.
* Code clean up across.

gsluthra reviewed (approved):
No comments from my side. Looks OK.

N0-man reviewed (commented):
- add .DS_Store files to .gitignore and cleanup
- we dont need jss-performance in the folder structure 
- What is the base for time interval e.g. pause(min = 10, max = 20)? How are we coming up with the time duration

samar-tw commented:
> * add .DS_Store files to .gitignore and cleanup
> * we dont need jss-performance in the folder structure
> * What is the base for time interval e.g. pause(min = 10, max = 20)? How are we coming up with the time duration

Updated .gitignore
Removed jss-performance folder
The pauses are completely on assumptions and is there hv a logical gap between the subsequent actions.
